docs/conf.py

```diff
@@ -49,9 +49,12 @@
 # intersphinx mapping
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/doc/stable/', None),
+    'numpydoc': ('https://numpydoc.readthedocs.io/en/stable/', None),
     'scipy': ('https://docs.scipy.org/doc/scipy/', None),
     'matplotlib': ('https://matplotlib.org/stable/', None),
-    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None)
+    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None),
+    'pytest': ('https://docs.pytest.org/en/stable/', None),
+    'coverage': ('https://coverage.readthedocs.io/en/latest/', None),
```

Comment:
```suggestion
    'coverage': ('https://coverage.readthedocs.io/en/stable/', None),
```

Reply:
I tried this, but its not working because it trys to fetch https://coverage.readthedocs.io/en/stable/objects.inv which is not working since https://coverage.readthedocs.io/en/stable/ is just a redirect to the latest stable version (e.g. https://coverage.readthedocs.io/en/7.5.0/) ...